### PR TITLE
gh-78955: Use user-selected color theme for Help => IDLE Doc

### PR DESCRIPTION
**Pull request by terryjreedy**
* Issue: gh-78955

### CODE REVIEW

**Comment by iritkatriel** [comment]
Updating the branch to kick off a new test run.

@terryjreedy This has been in "awaiting merge" for a few years now. What's the status?

**Comment by nineteendo** [comment]
@terryjreedy, this pull request is awaiting a merge for 5.5 years. Any updates?

**Comment by terryjreedy** [comment]
In https://github.com/python/cpython/issues/78955#issuecomment-1093800252 I noted the border color for the 2 `<pre>` (code) blocks on the page is seems to be fixed as black, which does not look good.  The current docs have visible borders in both light and dark mode.  I did not find anything else looking at tkinter and tk docs today.  If I do not find otherwise in a bit more research, I will take it as is.

File: Misc/NEWS.d/next/IDLE/2018-09-23-01-36-39.bpo-34774.VeM-X-.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Use user-selected color theme for Help => IDLE Help.
```
Comment (terryjreedy) [inline Misc/NEWS.d/next/IDLE/2018-09-23-01-36-39.bpo-34774.VeM-X-.rst]:
```suggestion
Use user-selected color theme for Help => IDLE Doc.
```

**Comment by terryjreedy** [comment]
I removed the border as it is fixed black.  I changed the pre block background to the text selectbackground.  Possibly confusing, but there is currently no way to get tk colors as rgb tuple that I know of.

**Pull request merged**